opam

```diff
@@ -34,9 +34,11 @@ remove: [
 depends: [
   "ocamlfind" {build}
   "re"
-  "sexplib" {>="109.53.00"}
+  "sexplib" {>= "113.24.00"}
```

Comment:
are you sure that this constraint for sexplib is needed? doesn't ppx_sexp_conv below suffice for `[@@deriving sexp]`?